base/markdown/render/latex.jl

```diff
@@ -136,5 +136,7 @@ latex(md) = sprint(latex, md)
 latexinline(md) = sprint(latexinline, md)
 latexesc(s) = sprint(latexesc, s)
 
+latexinline(io::IO, x) = writemime(io, MIME"text/plain"(), x)
```

Comment:
I think this should `latexesc`?

I have a feeling I intentionally removed this...

*The idea being that users/libraries should be defining there own latex rendering of their type rather fallthough to string/markdown, and it should raise if they haven't.*